From fe4d3d95ab0a264cbad9f19b16285002ba261179 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Thu, 21 Apr 2022 11:36:35 +0800
Subject: [PATCH 33/91] arm64: dts: rockchip: disable usb3 on quartz64

USB 3 ports on Quartz64 is of bad quality as it shares lines with SATA.
Disable usb3 support to prevent issues.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts b/arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts
index 0b191d8462ad..23287780d26a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts
@@ -789,6 +789,10 @@ &usb_host0_xhci {
 
 /* usb3 controller is muxed with sata1 */
 &usb_host1_xhci {
+	phys = <&usb2phy1_otg>;
+	phy-names = "usb2-phy";
+	extcon = <&usb2phy1>;
+	maximum-speed = "high-speed";
 	status = "okay";
 };
 
@@ -802,7 +806,7 @@ &usb2phy0_host {
 };
 
 &usb2phy0_otg {
-	phy-supply = <&vcc5v0_usb20_otg>;
+	phy-supply = <&vcc5v0_usb20_host>;
 	status = "okay";
 };
 
-- 
2.43.4

